### DISCUSSION TITLE
APU Start not possible

### DISCUSSION THREAD

**Discussion by abhovi** [flybywiresim/aircraft#4838] [Help]
While starting up from cold and dark the APU master switch works, but the APU starter switch does not work. When I hover over the switch I get: 'TT.cockpit.tooltips.apu.available'. But nothing happens. I don't understand what it means. Can anyone help me out?

**Comment by tracernz** [comment]
Which language do you use in MSFS?

**Reply by abhovi** [reply]
Dutch. Is that the cause?

**Reply by abhovi** [reply]
I switched over to English. The knob says Avail, but the APU does not start.

**Reply by tracernz** [reply]
> Dutch. Is that the cause?

We don't have a Dutch translation which might explain why the tooltip is not correct. Doesn't explain why the APU won't start though. If you have fuel in the left inner wing tank, both batteries on, and APU master is on, it should be able to start...

**Comment by abhovi** [comment]
I have to apologize. APU is working. It was just my stupidity.
Ab
